### ISSUE TITLE
Enhancement +Spell Damage

### ISSUE THREAD

**Issue by esandvik** [Muttley/foundryvtt-shadowdark#538]
Knight of St. Ydris uses a + to all damage  
There is currently bonuses.meleeDamageBonus, bonuses.rangedDamageBonus but no bonuses.spellDamageBonus attribute   
It also allows for future classes that may use a bonus to spell damage.

**Comment by Muttley** [comment]
The reason for this is that currently there's no concept of spell damage at all in the system.  Spells only perform success/failure rolls, damage is a manual thing.

**Comment by esandvik** [comment]
Yeah I saw that today when I started digging into the system.  So for now I think this can wait until that is added. But I do think that damage would be useful on the spell

**Comment by Muttley** [comment]
> But I do think that damage would be useful on the spell

Yes, definitely.

**Comment by deukalion-io** [comment]
I think this can be closed as it was addressed in #1275
